src/subclass/mod.rs

```diff
@@ -12,28 +12,60 @@
 //! ```rust
 //! #[macro_use]
 //! extern crate glib;
+//! #[macro_use]
+//! extern crate genum_derive;
```

Comment:
Maybe we should re-export that crate from the `glib` crate so it's easier available?

Reply:
What's the proper way to re-export such macro?
I tried this:
```rust
#[macro_use]
extern crate glib_macros;
pub use glib_macros::genum_derive;
```

but got:
```
error[E0432]: unresolved import `glib_macros::genum_derive`
  --> src/lib.rs:98:9
   |
98 | pub use glib_macros::genum_derive;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^ no `genum_derive` in the root
```

Reply:
`pub use glib_macros::GEnum` i think